utils/R/convert.input.R

```diff
@@ -9,7 +9,7 @@
 convert.input <- function(input.id, outfolder, formatname, mimetype, site.id, start_date, 
                           end_date, pkg, fcn, con = con, host, browndog, write = TRUE, 
                           format.vars, overwrite = FALSE, exact.dates = FALSE, 
-                          allow.conflicting.dates = TRUE, insert.new.file = FALSE,...) {
+                          allow.conflicting.dates = TRUE, insert.new.file = FALSE, met = NULL,...) {
```

Comment:
I can change it. I called it `met` because it comes from the met.process object created [here](https://github.com/PecanProject/pecan/blob/develop/modules/data.atmosphere/R/met.process.R#L47) and then is passed onto the .download.raw.met.module [here](https://github.com/PecanProject/pecan/blob/develop/modules/data.atmosphere/R/met.process.R#L160) and then to convert.inputs.

I'll rename the convert.input arg to `pattern` and change [this line](https://github.com/PecanProject/pecan/blob/release/1.5.0/modules/data.atmosphere/R/download.raw.met.module.R#L25) to `pattern = met`.